src/platforms/dialogflow/DialogflowEvent.ts

```diff
@@ -35,16 +35,15 @@ import { VoxaEvent } from "../../VoxaEvent";
 import { DialogflowIntent } from "./DialogflowIntent";
 import { DialogflowSession } from "./DialogflowSession";
 
-export interface IGoogle {
+export interface IDialogflow {
   conv: DialogflowConversation;
 }
 
 export class DialogflowEvent extends VoxaEvent {
   public rawEvent!: GoogleCloudDialogflowV2WebhookRequest;
   public session!: DialogflowSession;
-  public google!: IGoogle;
+  public dialogflow!: IDialogflow;
```

Comment:
not sure about this, maybe we should make `dialogflow` key an alias of `google` and deprecate the `google` key? 
Otherwise we're making it so projects with an older voxa version will have their code not work anymore